lib/cc/yaml/nodes/root.rb

```diff
@@ -16,6 +20,17 @@ def nested_warnings(*)
         def inspect
           "#<#{self.class.name}:#{super}>"
         end
+
+        def exclude?(path)
+          exclude_paths.present? &&
+            exclude_paths.match_any?(path)
+        end
+
+        def rate?(path)
+          ratings.nil? ||
+            ratings.paths.blank? ||
```

Comment:
Yeah, on the one hand there's risk of GodObject, but on the other there's risk of ShutgunSurgery since not having these delegators forces all callers to violate Demeter. We have pain with both of these smells in our current apps, so I'm not sure which is better to pre-avoid. Agree to see how it evolves; a "God Delegator" doesn't bother me if delegation is the primary responsibility of the object, which could make sense for `Root`.